setup.py

```diff
@@ -56,6 +57,7 @@
               'License :: OSI Approved :: BSD License',
               'Operating System :: POSIX :: Linux',
               'Programming Language :: Python :: 2.7',
+              'Programming Language :: Python :: 3',
```

Comment:
I'ts better to state the exact version, 3.6 seems to be the current stable.

Reply:
thanks fixed